spec/rubocop/target_ruby_spec.rb

```diff
@@ -308,33 +308,89 @@
             expect(target_ruby.version).to eq 2.7
           end
 
-          it 'sets target_ruby from required_ruby_version from inclusive range' do
+          it 'sets target_ruby from exacts string version' do
             content =
               <<-HEREDOC
                 Gem::Specification.new do |s|
                   s.name = 'test'
-                  s.required_ruby_version = '>= 3.0.0'
+                  s.required_ruby_version = '2.7.4'
                   s.licenses = ['MIT']
                 end
               HEREDOC
 
             create_file(gemspec_file_path, content)
-            expect(target_ruby.version).to eq 3.0
+            expect(target_ruby.version).to eq 2.7
           end
 
-          it 'sets default target_ruby from exclusive range' do
+          it 'sets target_ruby to default from unknown ruby' do
             content =
               <<-HEREDOC
                 Gem::Specification.new do |s|
                   s.name = 'test'
-                  s.required_ruby_version = '< 3.0.0'
+                  s.required_ruby_version = '2.3.0'
                   s.licenses = ['MIT']
                 end
               HEREDOC
 
             create_file(gemspec_file_path, content)
             expect(target_ruby.version).to eq default_version
           end
+
+          it 'sets target_ruby from inclusive range' do
+            content =
```

Comment:
No obligation, but consider using `let` or `before_each` to avoid duplicating all these lines so many times.